travis/push-to-pypi.sh

```diff
@@ -27,20 +23,18 @@ DEV_WHEEL_NAME="acc_provision-${TRAVIS_TAG}.dev${TRAVIS_BUILD_NUMBER}.tar.gz"
 DEV_TAG_NAME="acc_provision-${TRAVIS_TAG}.dev${TRAVIS_BUILD_NUMBER}"
 
 
-
-if [ -n "$PYPI_RELEASE" ] ; then
-    #twine upload --repository-url https://pypi.org/legacy/ -u ${PYPI_USER} -p ${PYPI_PASS} dist/$WHEEL_NAME
-    python setup.py sdist
-    twine upload -u ${PYPI_USER} -p ${PYPI_PASS} dist/$WHEEL_NAME
-    $SCRIPTS_DIR/push-to-cicd-status.sh "https://pypi.org/project/acc-provision/"${TRAVIS_TAG}"/#files" "${TAG_NAME}"
-elif [ -n "$TEST_PYPI_RELEASE" ] ; then
+if [ "$TRAVIS_COMMIT_AUTHOR" == "noiro-tagger" ] &&  [ -z "$SIGNED_EMAIL" ]; then
     #twine upload --repository-url https://test.pypi.org/legacy/ -u ${TEST_PYPI_USER} -p ${TEST_PYPI_PASS} dist/$WHEEL_NAME
```

Comment:
We still need to be able to push to pypi. Nothing should change in this part. The suggested changes are all for test-pypi.